README.md

```diff
@@ -12,6 +12,20 @@ This will do the following:
 
 ## Usage
 
+### Prerequisites
+
+1. Obtain a [GitHub personal access token][generate-token].
+2. Make sure the [token is available as the `GITHUB_AUTH` environment variable][export-token].
+  For instance:
+  ```bash
+  export GITHUB_AUTH=abc123def456
+  ```
+
+[generate-token]: https://github.com/settings/tokens/new?scopes=repo&description=GITHUB_AUTH+env+variable
+[export-token]: https://github.com/release-it/release-it/blob/master/docs/environment-variables.md
```

Comment:
Let's not link to the `release-it` docs here.  I think it risks confusion since they talk about `GITHUB_TOKEN` rather than `GITHUB_AUTH`.  I think the bash snippet you already have above is perfectly sufficient.